ci/docker/bosh-google-cpi-boshrelease/Dockerfile

```diff
@@ -50,6 +50,9 @@ RUN ruby-install ruby 2.1.2 && \
 # Install Bundler and BOSH CLI
 RUN /bin/bash -l -c "gem install bundler bosh_cli --no-ri --no-rdoc"
 
+# Install AWS CLI (used to upload stemcell)
+RUN pip install awscli
```

Comment:
Oops — your comment exposed a flaw in our PR (we pointed to a stale Docker image/tag). We're gonna update the PR now. Thanks for the keen eyes!